db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/SYSCOLUMNSRowFactory.java

```diff
@@ -60,17 +60,17 @@
 public class SYSCOLUMNSRowFactory extends CatalogRowFactory {
     public static final String		TABLENAME_STRING = "SYSCOLUMNS";
 
-    protected static final int		SYSCOLUMNS_COLUMN_COUNT = 13;
+    public static final int		SYSCOLUMNS_COLUMN_COUNT = 13;
 	/* Column #s for syscolumns (1 based) */
 
     //TABLEID is an obsolete name, it is better to use
     //REFERENCEID, but to make life easier you can use either
     protected static final int		SYSCOLUMNS_TABLEID = 1;
-    protected static final int		SYSCOLUMNS_REFERENCEID = 1;
-    protected static final int		SYSCOLUMNS_COLUMNNAME = 2;
-    protected static final int		SYSCOLUMNS_COLUMNNUMBER = 3;
+    public static final int		SYSCOLUMNS_REFERENCEID = 1;
+    public static final int		SYSCOLUMNS_COLUMNNAME = 2;
+    public static final int		SYSCOLUMNS_COLUMNNUMBER = 3;
     protected static final int		SYSCOLUMNS_STORAGECOLUMNNUMBER = 4;
-    protected static final int		SYSCOLUMNS_COLUMNDATATYPE = 5;
+    public static final int		SYSCOLUMNS_COLUMNDATATYPE = 5;
```

Comment:
(minor comment) tab/space alignment looks off

Reply:
I wouldn't fix it here since it would make this change even bigger, I can do it later in a more focused refactoring change.